docs/guides/api-development-strategy.md

```diff
@@ -49,12 +49,46 @@ The lifecycle is thus either:
 
 ## Current Future Flags
 
-Here's the current future flags in React Router v6 today:
+Here's the current future flags in React Router v6 today.
```

Comment:
Does this feel confusing?  Some flags are router-only and have nothing to do with rendering, others are render-only so I kept them split, but this does mean we export a `FutureConfig` from both `@remix-run/router` and `react-router`.